commerce-account-web/src/main/java/com/liferay/commerce/account/web/internal/frontend/CommerceAccountUserClayTable.java

```diff
@@ -233,11 +234,13 @@ public boolean isShowActionsMenu() {
 
 		Stream<Role> stream = roles.stream();
 
-		return stream.map(
+		String[] rolesArray = stream.map(
 			Role::getName
 		).toArray(
 			String[]::new
 		);
+
+		return StringUtil.merge(rolesArray, StringPool.COMMA_AND_SPACE);
```

Comment:
This method seems unnecessarily complex... loop (above) + stream + `StringUtil.merge`. We could do it all with stream operations (which might be pretty nice), or do loop + `ListUtil.toString` (which would also be fine). Either way that'd simplify the method and make it perform a little better.

Reply:
I think I remember fixing something very similar or maybe the same spot in a newer PR. How's this look in place of the stream and old return:
`return ListUtil.toString(roles, Role.NAME_ACCESSOR, StringPool.COMMA_AND_SPACE);`

Reply:
Looks great!